doc/development/developing-sql-dialect/integration_testing_with_containers.md

```diff
@@ -21,99 +21,29 @@ What you need is, for each source database:
 
 ## Preparing Integration Test
 
-1. In order to run the integration test automated, edit the  [Travis CI integration test configuration file](../../../integration-test-data/integration-test-travis.yaml) and add your new database.
+1. In order to run the integration test automated, add the test to the includes list of the `maven-failsafe-plugin` in the [pom file](../../../pom.xml).
 2. Provide a JDBC driver JAR for the source database.
 3. Add a new Integration Test class for you database
 
-### Add Your Database to the Test Configuration
-Set the following properties for your database:
-
-| Configuration property   | Explanation | Example |
-|--------------------------|-------------|---------|
-| `runIntegrationTests`    | enable/disable your test | `true` |
-| `jdbcDriverPath`         | path to the JDBC driver in bucketFS | `/buckets/bfsdefault/default/drivers/jdbc/POSTGRESQL/postgresql-42.2.5.jar`|
-| `connectionString`       | connection string to connect to the source database from the integration test system, so use the exposed port | `jdbc:postgresql://localhost:45432/postgres` |
-| `user`                   | the database user | |
-| `password`               | password for the database user| |
-| `dockerImage`            | name of the docker image for the source database | |
-| `dockerImageVersion`     | version of the used docker image | `latest` |
-| `dockerPortMapping`      | docker port mapping `<external_db_port>:<internal_db_port>` | `45432:5432` |
-| `dockerName`             | name for the docker container < `testpg` |
-| `dockerConnectionString` | connection string to connect to the source database from the EXASOL docker container. Use the constant `DBHOST` as hostname, this will be set to the actual internal docker network IP by the integration test script during runtime | `jdbc:postgresql://DBHOST:5432/postgres` |
-
 ### Provide JDBC drivers for the Source Database
 
-The JDBC drivers are automatically deployed during the test. You have to create a directory for the jdbc driver under integration-test-data/drivers. The folder contains the driver jar file(s) and a config file. See the [PostgreSQL config](../../../integration-test-data/drivers/POSTGRESQL/settings.cfg) for an example.
-
+The JDBC drivers are automatically deployed during the test. You have to create a directory for the jdbc driver under `src/test/resources/integration/driver`. 
+The folder contains the driver jar file(s) and a `settings.cfg` file (for any integration test except Exasol and Postgres).
 In order to connect to the source database from your integration test you also have to add the jdbc driver dependency to the [POM](../../../pom.xml) scope verify.
```

Comment:
Add an explanation that we can only deliver drivers where the license allows redistribution. Users should understand, why it is so cumbersome to get the test running.

Reply:
done